/home/Plex/Bots/Niall/niall.js:27
let tmp=fs.readdirSync("./core".filter(file => file.endsWith(".js"));
                                                                   ^

SyntaxError: missing ) after argument list
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Niall@1.2.0 start: `node .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Niall@1.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/plex/.npm/_logs/2022-03-01T20_15_26_649Z-debug.log
/home/Plex/Bots/Niall/niall.js:28
for (const file of tmp) global[file.slice(0,-3)]=require(`./core/${file}`),plg);
                                                                              ^

SyntaxError: Unexpected token ')'
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Niall@1.2.0 start: `node .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Niall@1.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/plex/.npm/_logs/2022-03-01T20_17_01_201Z-debug.log
internal/fs/utils.js:332
    throw err;
    ^

Error: ENOENT: no such file or directory, open './birthday.csv'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at Import (/home/Plex/Bots/Niall/modules/Birthday/bday.js:26:14)
    at Object.<anonymous> (/home/Plex/Bots/Niall/modules/Birthday/bday.js:115:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './birthday.csv'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Niall@1.2.0 start: `node .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Niall@1.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/plex/.npm/_logs/2022-03-01T20_29_15_374Z-debug.log
internal/fs/utils.js:332
    throw err;
    ^

Error: ENOENT: no such file or directory, open './darebee.csv'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at parseCSV (/home/Plex/Bots/Niall/modules/DB/darebee.js:9:18)
    at Object.<anonymous> (/home/Plex/Bots/Niall/modules/DB/darebee.js:426:10)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './darebee.csv'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Niall@1.2.0 start: `node .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Niall@1.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/plex/.npm/_logs/2022-03-01T20_31_00_321Z-debug.log
internal/fs/utils.js:332
    throw err;
    ^

Error: ENOENT: no such file or directory, open '../temp.csv'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at parseCSV (/home/Plex/Bots/Niall/core/temp.js:5:18)
    at Object.<anonymous> (/home/Plex/Bots/Niall/core/temp.js:54:3)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '../temp.csv'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Niall@1.2.0 start: `node .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Niall@1.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/plex/.npm/_logs/2022-03-01T20_32_31_931Z-debug.log
